plugins/scaffolder-actions/scaffolder-backend-module-http-request/src/actions/run/backstageRequest.ts

```diff
@@ -103,11 +110,14 @@ export function createHttpBackstageAction(options: {
       const { input } = ctx;
       const token = ctx.secrets?.backstageToken;
       const { method, params } = input;
+      const logging = input.logging ?? true;
       const url = await generateBackstageUrl(discovery, input.path);
 
-      ctx.logger.info(
-        `Creating ${method} request with ${this.id} scaffolder action against ${input.path}`,
-      );
+      if (logging === true) {
```

Comment:
Shorthand is fine here since it's a boolean
```suggestion
      if (logging) {
```

Reply:
Good catch! Had some trouble with that and forgot to remove it after